### ISSUE TITLE
Material processing units(All of them) broken (kinda)

### ISSUE THREAD

**Issue by Joshuawood1998** [UmbraSpaceIndustries/MKS#1388]
They are saying "Install(none)" on all of the sections for processors. 

So i cant change what they are processing/tell what they are processing they seem to be stuck on LFO from ORE. Tried all the different versions i can find only adding certain parts updating community resource pack all sorts.

**Comment by boiler1** [comment]
I am running 1.4.1 with MKS .55 and I do not see the problem you are having.  Are you sure you have the mod installed correctly?

**Comment by Joshuawood1998** [comment]
I'm running 1.3.1 and using ckan. I swapped the relevant gamedata folders with several versions of MKS some of them broke it even more (to be expected) None of them fixed the problem. I tried many combinations of old files that relate directly to the parts but couldn't get it to work then either.

Even tried a brand new version of 1.3.1 ksp with no extra mods just MKS and installing with ckan and it didn't work then either.
